code/game/machinery/washing_machine.dm

```diff
@@ -268,6 +268,13 @@ GLOBAL_LIST_INIT(dye_registry, list(
 		slowdown = SHOES_SLOWDOWN
 		new /obj/item/restraints/handcuffs(loc)
 	..()
+	
+/obj/item/clothing/shoes/clown_shoes/machine_wash(obj/machinery/washing_machine/WM)
+	if(slowdown==15)
```

Comment:
```suggestion
	if(istype(/obj/item/clothing/shoes/clown_shoes/prison))
```

Reply:
):<

![Immagine](https://user-images.githubusercontent.com/5155917/89666542-d4a0ce80-d8da-11ea-847b-9b04a4f7dd9e.png)

Reply:
oh, whoops, I set that up wrong